steps/jandi-message/0.1.0/step.yml

```diff
@@ -0,0 +1,117 @@
+title: Send a Jandi message
+summary: Send a [Jandi](http://bit.ly/2FiXoc5) message to a topick.
+description: |
+  Send a [Jandi](http://bit.ly/2FiXoc5) message to a topick.
+  You can setup **Incoming WebHook integration**.
+  To register an **Incoming WebHook integration** visit http://bit.ly/31Bxeeh
+
+  *Don't forget to enable this Step to **Always run** if
+  you want to get notified about failed builds as well!*
+
+  This Step is inspired from [Send a slck message step](https://github.com/bitrise-steplib/steps-slack-message).
```

Comment:
typo `slck` -> `slack`, but I would not mention this, as it is not relevant for a user.